framework/Models/PostProcessors/CrossValidation.py

```diff
@@ -329,7 +327,7 @@ def run(self, inputIn):
           scoreDict[varName] = np.atleast_1d(np.mean(np.atleast_1d(metricValues)))
       return scoreDict
 
-  def collectOutput(self,finishedJob, output):
+  def collectOutput(self,finishedJob, output, options=None):
```

Comment:
options for?

Reply:
This options is from Model entity that you added. I keep it here for consistency. We can remove it if you agree.

Reply:
I have removed the options.